sdk/python/tests/integration/feature_repos/test_repo_configuration.py

```diff
@@ -158,6 +158,32 @@ def vary_infer_event_timestamp_col(
     return new_configs
 
 
+def vary_redis_online_store(configs: List[TestRepoConfig],) -> List[TestRepoConfig]:
+    new_configs = []
+    redis_config = {"type": "redis", "connection_string": "localhost:6379,db=0"}
+    for c in configs:
+        if (
+            "FileDataSourceCreator" in c.offline_store_creator
```

Comment:
also since this is the first method that processes the static list of configs, are these if statements even needed? you basically are taking every existing combo and adding redis

Reply:
Sure, I'll just make the redis config static and add to th list